app/src/main/res/values/strings.xml

```diff
@@ -2,8 +2,8 @@
     <string name="app_name">Brainiacs</string>
 
     <!-- Strings related to login -->
-    <string name="prompt_email">Email</string>
-    <string name="prompt_password">Password (optional)</string>
+    <string name="prompt_email">Username</string>
```

Comment:
Change the name portion of this from prompt_email to prompt_username to better reflect the representation.